src/box/lua/load_cfg.lua

```diff
@@ -423,6 +399,17 @@ local dynamic_cfg_modules = {
             flightrec_requests_max_res_size = true,
         },
     },
+    log = {
+        cfg = log.internal.box_api_cfg,
+        settings = log.cfg,
```

Comment:
Bad naming: "settings" is a synonym to "options", which looks confusing. Discussed f2f and agreed that `log.cfg` should update `box.internal.cfg` directly, which will be then used on first call to `box.cfg`.

Reply:
Fixed.